apps/omg_watcher/lib/omg_watcher/sync_supervisor.ex

```diff
@@ -131,7 +131,7 @@ defmodule OMG.Watcher.SyncSupervisor do
         service_name: :depositor,
         synced_height_update_key: :last_depositor_eth_height,
         get_events_callback: &EthereumEventAggregator.deposit_created/2,
-        process_events_callback: &OMG.State.deposit/1
+        process_events_callback: &Watcher.Depositor.new_deposits/1
```

Comment:
I don't see a particular reason why this `Watcher.Depositor` a new module, where all other events are being handled in place (where they were before).

we should put 
```
if Code.ensure_loaded?(OMG.WatcherInfo.DB.EthEvent),
      do: Kernel.apply(OMG.WatcherInfo.DB.EthEvent, :insert_deposits!, [deposits])
```
into `OMG.State.deposit/1` for consistency with everything else